data/json/monsters/nether.json

```
@@ -1970,5 +1970,140 @@
+    "morale": 100,
+    "aggression": 5,
+    "symbol": "v",
+    "color": "light_gray",
+    "bodytype": "blob",
+    "harvest": "flesh_plant",
+    "flags": [ "GRABS", "HEARS" ]
+  },
+  {
+    "id": "mon_nether_unviable_stomper",
+    "type": "MONSTER",
+    "name": { "str": "eldritch stomper" },
+    "description": "Mostly covered in wet-looking, long strands that remind you firmly of hair, this abomination has a single but very pronounced leg.  It keeps its balance with its odd hair on the rear.  Yet it seems to shrivel and die with each gust of wind it is hit with.",
+    "species": "ABERRATION",
+    "hp": 80,
+    "speed": 80,
+    "diff": 6,
+    "weight": "30 kg",
+    "volume": "50 L",
+    "default_faction": "nether",
+    "anger_triggers": [ "HURT", "PLAYER_CLOSE" ],
+    "morale": 80,
+    "aggression": 80,
+    "symbol": "e",
+    "color": "blue",
+    "bodytype": "kangaroo",
+    "harvest": "mutant_animal_noskin",
+    "special_attacks": [ { "id": "mutant_hoof" } ],
+    "zombify_into": "mon_meat_cocoon_med",
+    "flags": [ "BASHES", "HEARS", "SEES", "LOUDMOVES" ]
+  },
+  {
+    "id": "mon_nether_unviable_bug",
+    "type": "MONSTER",
+    "name": { "str": "strange bug" },
+    "description": "This small, three legged bug-like creature curiously wriggles its 5 antennae.  Its body somewhat reminds you of a stink bug.  It looks like a fish out of water and is getting weaker by the minute.",
+    "species": "INSECT",
+    "hp": 20,
+    "speed": 110,
+    "diff": 2,
+    "weight": "2 kg",
+    "volume": "5 L",
+    "default_faction": "nether",
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "melee_damage": [ { "damage_type": "cut", "amount": 2 } ],
+    "anger_triggers": [ "HURT", "FRIEND_DIED", "PLAYER_CLOSE" ],
+    "morale": 30,
+    "aggression": -5,
+    "symbol": "a",
+    "color": "pink",
+    "bodytype": "insect",
+    "harvest": "arachnid_beetle",
+    "special_attacks": [ { "type": "bite", "attack_upper": false } ],
+    "zombify_into": "mon_meat_cocoon_tiny",
+    "flags": [ "SEES", "HEARS" ]
+  },
+  {
+    "id": "mon_nether_unviable_chunk",
+    "type": "MONSTER",
+    "name": { "str": "chunk of alien flesh", "str_pl": "chunks of alien flesh" },
+    "description": "A writhing mass of strange flesh and something that oddly resembles tentacles.  It is getting more pale by the minute.",
+    "species": "NETHER",
+    "hp": 20,
+    "diff": 2,
+    "weight": "2 kg",
+    "volume": "5 L",
+    "symbol": "v",
+    "color": "pink",
+    "default_faction": "nether",
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "melee_damage": [ { "damage_type": "bash", "amount": 1 } ],
+    "special_attacks": [ { "id": "grab" } ],
+    "grab_strength": 3,
+    "anger_triggers": [ "HURT", "PLAYER_CLOSE" ],
+    "morale": 100,
+    "aggression": 20,
+    "bodytype": "blob",
+    "harvest": "mutant_tiny",
+    "zombify_into": "mon_meat_cocoon_tiny",
```

```suggestion
    "zombify_into": "mon_meat_cocoon_tiny",
```
I'm going ahead and merging this but I could go either way on whether these nether creatures zombify into anything or not.